test/unit/serialization/PortableObjects.js

```diff
@@ -25,13 +27,13 @@ class InnerPortable {
     }
 
     readPortable(reader) {
-        this.p1 = reader.readString('p1');
-        this.p2 = reader.readString('p2');
+        this.p1 = TestUtil.readStringFromReader(reader, 'p1');
```

Comment:
Same, why change these as these are not in backward_compatible folder

Reply:
These are used in multiple places including backward compatible test suite